index.js

```diff
@@ -133,11 +133,13 @@ var createAssertion = function createAssertion(type, message, wrappers, block, m
 	if (descriptors.length === 0) {
 		throw new RangeError(inspect(type) + ' called with no wrappers defined');
 	}
-	var describeMsgs = [];
-	forEach(wrappers, function (wrapper) {
-		checkThis(wrapper);
-		describeMsgs.push(getThisDescription(wrapper));
```

Comment:
hmm - this means it's mutating the descriptor object the user passes in; and I'm concerned that the uses of the `concatThis` helper (in `skip` and `only`) without `extend` will mean that those places aren't persisting the description.

Could we fix this without mutating `descriptor` or changing how descriptions are stored?